src/Messages/Messages.module.css

```diff
@@ -0,0 +1,191 @@
+.messagesPageContainer {
+  max-width: 935px;
+  margin: 80px auto;
+  display: flex;
+}
+
+.userSection {
+  display: flex;
+  width: 350px;
+  border-top: 1px solid #dbdbdb;
+  border-left: 1px solid #dbdbdb;
+  border-right: 1px solid #dbdbdb;
+  padding: 0;
+  margin: 0;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+}
+.sendMessageIcon {
+  padding-right: 20px;
+}
+.usernameIconSection {
+  display: flex;
+  margin: 30px 20px 30px 120px;
+  background-color: #fff;
+}
+.arrowDownIcon {
+  transform: rotate(180deg);
+  align-self: center;
+  padding: 0 10px;
+}
+.sendersSection {
+  width: 350px;
+  border: 1px solid #dbdbdb;
+  overflow: hidden auto;
+  height: 50%;
```

Comment:
hmmmm. 50% filling the viewport means the container (`leftSection`) is too big. That's why you have empty scrolling below on the screen. Make this 100%, then make the container smaller. I'd make the container of this div like this

```css
.leftSection {
	/* screen height - margin top - margin bottom */
	height: calc( 100vh - 80px - 80px); 
}
```

If you do this for both sides (left and right) you'll get the same responsiveness as IG.

Reply:
On it